hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -124,6 +124,23 @@ execute_command_in_container(){
   set +e
 }
 
+## @description Stop a list of named containers
+## @param       List of container names, eg datanode_1 datanode_2
+stop_containers() {
+  set -e
+  docker-compose -f "$COMPOSE_FILE" stop $@
```

Comment:
```suggestion
  docker-compose -f "$COMPOSE_FILE" --no-ansi stop $@
```

Please add `--no-ansi` to avoid funky characters in the output:

```
2020-02-26T17:01:00.9125703Z [3BStarting datanode_1 ... 
2020-02-26T17:01:00.9127089Z Starting datanode_2 ... 
2020-02-26T17:01:00.9128127Z Starting datanode_3 ... 
2020-02-26T17:01:01.4869851Z [1A[2K
2020-02-26T17:01:01.4870604Z Starting datanode_3 ... [32mdone[0m
2020-02-26T17:01:01.5357903Z [1B[3A[2K
2020-02-26T17:01:01.5359166Z Starting datanode_1 ... [32mdone[0m
2020-02-26T17:01:01.6387370Z [3B[2A[2K
```
```